helm/alfresco-content-services/values.yaml

```diff
@@ -739,6 +734,8 @@ alfresco-sync-service:
   syncservice:
     enabled : true
 
+global:
```

Comment:
Did you mean to move this up or was it a typo?

Reply:
It was intentional to try to distinguish existing:

```
global.alfrescoRegistryPullSecrets
global.strategy.*
global.registryPullSecrets
```

from new:

`global.ai.enabled`